Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3683,7 +3683,7 @@ ErrorOr<RefPtr<StyleValue>> Parser::parse_dimension_value(ComponentValue const&
     // 2) It's a 0.
     // We handle case 1 here. Case 2 is handled by NumericStyleValue pretending to be a LengthStyleValue if it is 0.
```

Comment:
Ah, this comment says I missed this when I changed parsing. We no longer do the pretending. Apparently no layout tests happened to rely on this. 🤔